lib/common.c

```diff
@@ -43,8 +43,14 @@
 #include <stdlib.h>
 #include <string.h>
 #include <sys/mman.h>
+#include <sys/resource.h>
 #include <unistd.h>
 
+/* Maximum required file descriptors per core */
+#define MAX_FD_PER_CORE 4
```

Comment:
The MAX_FD_PER_CORE should be 5.

1 fd to read/write msr
2 fd to obtain IPC value
1 fd to obtain LLC miss value
1 fd to obtain LLC ref value

Reply:
Done